src/main/java/Duke.java

```diff
@@ -1,29 +1,28 @@
 import java.util.Scanner;
 
 public class Duke {
-
+    static final String LOGO = "   _____                .__   \n" +
+            "  /  _  \\ ___  ___ ____ |  |  \n" +
+            " /  /_\\  \\\\  \\/  // __ \\|  |  \n" +
+            "/    |    \\>    <\\  ___/|  |__\n" +
+            "\\____|__  /__/\\_ \\\\___  >____/\n";
     public static void greetUser(){
-        String greeting = "____________________________________________________________" + System.lineSeparator()
```

Comment:
Good modification of the usage of the new line syntax. "line.seperator()" can indeed take up a lot of code space.